src/main/resources/org/craftercms/studio/api/v2/dal/AuditDAO.xml

```diff
@@ -280,6 +285,18 @@
         AND origin = 'API'
     </select>
 
+    <select id="getCommitAuthors" parameterType="java.util.Map" resultMap="CommitAuthorMap">
+        SELECT a.commit_id, u.*
+        FROM audit a
+        INNER JOIN user u on u.username = a.actor_id
+        WHERE a.commit_id IN
```

Comment:
Should we filter by site id? Can we have the same commit id in different sites if cloning from the same remote?

Reply:
@jmendeza It's best to filter with the site id so I updated to include this